docs/migration_guide.md

```diff
@@ -1,6 +1,6 @@
 # Harbor upgrade and database migration guide
 
-Refer to [change log](../tools/migration/changelog.md) to find out whether there is any change in the database. If there is, you should go through the database migration process. 
+Refer to [change log](../tools/migration/changelog.md) to find out whether there is any change in the database or harbor.cfg. If there is, you should go through the database migration process.
 
 **NOTE:**
 - From v1.6.0 onwards, Harbor migrated from MariaDB to Postgresql, combining Harbor, Notary and Clair databases into one. 
```

Comment:
We set a assumed directory, and updated the commands to be consistent.

For steps 4 and 5, i think there is confusion on which of steps should be taken, as it looks like you can perform multiple different ways of migration depending on your situation.